src/DataCollector/MultiAuthCollector.php

```diff
@@ -115,7 +115,7 @@ protected function getUserInformation($user = null)
         // The default auth identifer is the ID number, which isn't all that
         // useful. Try username and email.
         $identifier = $user instanceof Authenticatable ? $user->getAuthIdentifier() : $user->id;
-        if (is_numeric($identifier)) {
+        if (is_numeric($identifier) || Str::isUuid($identifier)) {
```

Comment:
You forgot `ulid`, must be:
```php
if (is_numeric($identifier) || Str::isUuid($identifier) || Str::isUlid($identifier)) {
```
[https://github.com/laravel/framework/src/Illuminate/Support/Str.php#L533-L540](https://github.com/laravel/framework/blob/3141ef2056213854abb62e2f6f4127525c3fc050/src/Illuminate/Support/Str.php#L533-L540)